src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css

```diff
@@ -9,7 +9,7 @@
 	position: absolute;
 	top: 0;
 	right: 18px;
-	width: 220px;
+	min-width: 360px;
```

Comment:
Changes to how the width work are necessary? More to verify if we change it up

Reply:
yeah looks like that wasn't necessary